### PR TITLE
ITL: Add CheckCommand ssl_cert, fix ssl attributes

### PR DESCRIPTION
**Pull request by rhopfer**
Allows to use ssl_cert_valid_days_warn without ssl_cert_valid_days_critical which is 0 for this case.
Include a CheckCommand for check_ssl_cert which is part of nagios-plugins-contrib in debian.

### CODE REVIEW

File: itl/command-plugins.conf [hunk 1]
```diff
@@ -275,15 +275,35 @@ object CheckCommand "ssl" {
...
+			description = "Seconds before connection times out (default: 10)"
+		}
+		"-D" = {{
+			if (macro("$ssl_cert_valid_days_warn$")) {
+				if (macro("$ssl_cert_valid_days_critical$")) {
+					return macro("$ssl_cert_valid_days_warn$,$ssl_cert_valid_days_critical$")
+				} else {
+					return macro("$ssl_cert_valid_days_warn$")
+				}
+			}
```
Comment (dnsmichi) [inline itl/command-plugins.conf]: I'd use local variables here, and save 2 macro() calls which are rather expensive.

**Comment by dnsmichi** [comment]
Thanks 👍 Can you please add documentation for the changes and the new CheckCommand too?

**Comment by rhopfer** [comment]
Sure! I will also add some options for the current version of check_ssl_cert.

**Comment by rhopfer** [comment]
I added now most of the options for the current version of check_ssl_cert to the check command. Unfortunatly my english is not so good, so please check the documentation.

**Comment by dnsmichi** [comment]
One last thing - check_ssl_cert is not part of the monitoring plugins, but a contributed CheckCommand definition. Can you please move that accordingly, i.e. put ssl-cert into the "web" section? TIA.

**Comment by rhopfer** [comment]
Done. I also brought the plugins in this section to alphabetical order.

**Comment by dnsmichi** [comment]
Oh, cool, you were faster than me. Thanks 👍

**Comment by dnsmichi** [comment]
Note: I've renamed the CheckCommand `cert` to `ssl_cert` and adopted its custom attributes. This follows our plugin and argument style guide. 

I've also rebased the branch and force pushed your repository to cleanly merge this pull request.

Please note that you should add your changes in local feature branches. This should be the base for all PRs you send, and allows you to keep a clean master branch. More details can be found here: https://github.com/Icinga/icinga2/blob/master/CONTRIBUTING.md

Thanks for your contribution 👍

**Pull request merged**